.misc/deploy.pypi.sh

```diff
@@ -4,18 +4,22 @@
 set -e
 
 # Get environment variables, readily decrypted by rultor
-source ../rultor_secrets.sh
+#source ../rultor_secrets.sh
 
 # Ship it!
 echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
```

Comment:
no I mean the name `twine_output.txt` itself may be stored in a variable. Not sure if that makes sense, that's why it's up to you ;)

> Also, do you mean that I store the output in a variable rather that in a file? That's what I did initially, but jayvdb suggested using a file instead.

just curious, why actually? I mean less I/O, or not?